libraries/shared/watcher/storage_watcher.go

```diff
@@ -43,12 +45,17 @@ func NewStorageWatcher(fetcher fetcher.IStorageFetcher, db *postgres.DB) Storage
 	queue := storage.NewStorageQueue(db)
 	return StorageWatcher{
 		db:             db,
+		diffSource:     "csv",
```

Comment:
I think that with the refactor in 8e8468f this is no longer as big of a deal since the only place the strings are used is in the commands to decide which `StorageWatcher` to create. 🤷‍♀